tests/functional/codeartifact/test_codeartifact_login.py

```diff
@@ -40,7 +40,7 @@ def setUp(self):
         self.pypi_rc_path_mock = self.pypi_rc_path_patch.start()
         self.pypi_rc_path_mock.return_value = self.test_pypi_rc_path
 
-        self.subprocess_patch = mock.patch('subprocess.check_call')
+        self.subprocess_patch = mock.patch('subprocess.run')
         self.subprocess_mock = self.subprocess_patch.start()
         self.subprocess_check_output_patch = mock.patch(
```

Comment:
We should be able to remove this patch I'm assuming with our refactoring?

Reply:
No, nuget and dotnet still use `check_output` to capture responses from list nuget source. I didn't want to change that because it is specific to nuget and dotnet and seems not in the scope of this pr.

Reply:
Make sense. Let's keep it as is then.